app/assess/templates/macros/fund_dashboard_summary.html

```diff
@@ -53,6 +53,13 @@ <h3 class="govuk-body govuk_summary-card__title govuk-!-font-size-24">{{round_na
             View all {% if is_active_status %}active{% else %}
             closed{% endif %} assessments
         </a>
+        {% if round_application_fields_download_available and access_controller.is_lead_assessor%}
+        </br>
+        </br>
+        <a class="govuk-link" data-qa="dashboard_export" href="{{ export_href }}">
```

Comment:
Hmm, `data-qa` looks like something added for @rayhan15243's automated tests.  Duplicating it may cause issues?  Just a heads up

Reply:
removed this just to be sure